transaction_test.go

```diff
@@ -283,12 +268,10 @@ func TestLogCallback(t *testing.T) {
 			Variable:     variables.UniqueID,
 		},
 	})
-	if buffer == "" && strings.Contains(buffer, tx.ID) {
-		t.Error("failed to call error log callback")
-	}
-	if err := tx.Clean(); err != nil {
-		t.Error(err)
-	}
+	require.Empty(t, buffer)
```

Comment:
We should revisit this test @fzipi @jptosso. Original assertion asks for buffer to be empty and then to contains a value.